sources/chgov.json

```diff
@@ -4,9 +4,9 @@
     "fullname": "Swiss Confederation", 
```

Comment:
Please change the fullname to `Swiss Topo` (it's their version of USGS)

Reply:
Changed in most recent commit.